development/docker-compose/docker-compose.yaml

```diff
@@ -16,18 +17,18 @@ x-common-variables: &common-variables
   eureka.instance.initialStatus: STARTING
   management.endpoints.web.exposure.include: '*'
   management.endpoints.enabled-by-default: true
-  management.security.roles: ANONYMOUS,ACTUATOR,USER
+  management.security.roles: ANONYMOUS,ACTUATOR
   management.endpoint.env.show-values: ALWAYS
+  management.endpoint.env.post.enabled: true
```

Comment:
this enables changing configuration properties in runtime without restarting service uing the actuator endpoints